src/assets/constants.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-unused-vars */
+const getPageHeight = () => {
```

Comment:
This file is weird. `constants` is a weird name for it to contain the `getPageHeight` function. `shared` is a better name. Let's rename it to that.

Reply:
`constants` is renamed to `shared`.